core/common/src/TimeZone.kt

```diff
@@ -104,6 +104,19 @@ public expect fun TimeZone.offsetAt(instant: Instant): ZoneOffset
  */
 public expect fun Instant.toLocalDateTime(timeZone: TimeZone): LocalDateTime
 
+/**
+ * Return a civil date value that this instant has in the specified [timeZone].
+ *
+ * Note that while this conversion is unambiguous, the inverse ([LocalDateTime.toInstant])
+ * is not necessary so.
+ *
+ * @see Instant.toLocalDateTime
+ * @see LocalDateTime.toInstant
+ * @see Instant.offsetIn
+ * @throws DateTimeArithmeticException if this value is too large to fit in [LocalDateTime].
+ */
+public expect fun Instant.toLocalDate(timeZone: TimeZone): LocalDate
```

Comment:
Exactly this is what I was looking for at the beginning to print only the date without the time

Using toLocalDateTime it would be: `2020-12-17T21:20:26.883`
And using toLocalDate it would be: `2020-12-17`

So, I could change the implementation that this code that you are describing to decouple the use cases.

Reply:
Then there could be 2 ways: either change the implementations of the targets using what you comment or just leaving this new line in the common code. I take the 2nd way